<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefInput</class>
 <widget class="QWidget" name="PrefInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/res.qrc">
    <normaloff>:/icons/default/input_devices.png</normaloff>:/icons/default/input_devices.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabKeyboard">
      <attribute name="title">
       <string>Keyboard</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMouse">
      <attribute name="title">
       <string>Mouse</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="mouseButtonsGrid">
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Middle click</string>
           </property>
           <property name="buddy">
            <cstring>middle_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="xbutton2_click_combo"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="left_click_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="textLabel1_16">
           <property name="text">
            <string>Button functions:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="textLabel3_5">
           <property name="text">
            <string>Right click</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>right_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X Button 1</string>
           </property>
           <property name="buddy">
            <cstring>xbutton1_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="middle_click_combo"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="double_click_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>X Button 2</string>
           </property>
           <property name="buddy">
            <cstring>xbutton2_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="right_click_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel3_4">
           <property name="text">
            <string>Double click</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>double_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel2_6">
           <property name="text">
            <string>Left click</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>left_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="xbutton1_click_combo"/>
         </item>
         <item row="0" column="2" rowspan="7">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>293</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/res.qrc"/>
 </resources>
 <connections/>
</ui>
